src/Security/WebsiteCop.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Application\Security;
+
+use Application\Exception\AppException;
+use Application\Exception\BlogException;
+use Application\Exception\SecurityException;
+use Model\Entity\ConnectionTry;
+use Model\Manager\ConnectionTryManager;
+
+class WebsiteCop
```

Comment:
Not sure to understand what this class does with only the name... It could be great to give a more explicit name or at least a small comment to explain what it does

Reply:
Done! I create 2 classes : CsrfProtector and BruteForceProtector.